### PR TITLE
Clean up version constraints

### PR DESCRIPTION
**Pull request by pkoenig10**

### CODE REVIEW

File: versions.props [hunk 1]
```diff
@@ -46,11 +48,3 @@ javax.annotation:javax.annotation-api = 1.3.2
 javax.ws.rs:javax.ws.rs-api = 2.1.1
 javax.inject:javax.inject = 1
 
-# dependency-upgrader:OFF
-
-com.fasterxml.jackson.*:* = 2.13.1
-
-# Pin to same version as spark
-io.dropwizard.metrics:metrics-core = 3.2.6
```
Comment (carterkozak) [inline versions.props:54]: When a new dropwizard-metrics version is released, we don't want this repo to to cause everything else to resolve it upward -- that has historically caused issues for spark.

Comment (pkoenig10) [reply versions.props:54]: But that already happens - we're currently using a 4.x version which is being bumped by transitive dependencies. So this constraint is just confusing.

Comment (carterkozak) [reply versions.props:54]: the dw-metrics dep should ideally be produced by tritium, however at the limit we'd end up splitting out that dependency into a separate excavator if we ran into that sort of api split again

**Comment by carterkozak** [comment]
:+1: you'll need to rebase in order to merge

**Comment by pkoenig10** [comment]
@carterkozak not sure why your 👍 isn't satisfying policy-bot

**Review by carterkozak — APPROVED** [review]
I think the policy files were updated recently to invalidate on push

**Pull request merged**